HelpSource/Classes/Char.schelp

```diff
@@ -4,25 +4,36 @@ categories:: Core
 
 description::
 Chars may be written as literals using the $ sign. For example $a, $b, $c.
-See link::Reference/Literals::
+Some special characters can be expressed as literals using escape sequences (for example, code::$\n:: for newline).
+See link::Reference/Literals#Characters:: for more information.
 
 Chars may be created from link::Classes/Integer::s using the methods link::Classes/Integer#-asAscii:: and link::Classes/Integer#-asDigit::.
 
 classmethods::
 
 method::nl
-Returns code::($\n)::
-method::ff
-Returns code::($\f)::
+Newline code::($\n)::.
+
+method::ret
+Carriage return code::($\r)::.
+
 method::tab
-Returns code::($\t)::
+Horizontal tab code::($\t)::.
+
+method::ff
+Form feed code::($\f)::.
+
+method::vtab
+Vertical tab code::($\v)::.
+
 method::space
-Returns code::($ )::
+Single space code::($ )::.
+
 method::comma
-Returns code::($\,)::
+Comma code::($,)::.
+
 method::bullet
-Returns a bullet character (•) in SuperCollider.app on macOS, but a simple asterisk (*) on other frontends.
-This method is not recommended, since it's actually not cross-platform.
```

Comment:
OK! After this PR clears I'll make a new one to deprecate it. I want to keep this PR focused on updating documentation. Thanks for the suggestion, I was wondering about this while I was working on it. :)